core/src/replay_stage.rs

```diff
@@ -1697,6 +1701,41 @@ impl ReplayStage {
                     replay_vote_sender,
                     verify_recyclers,
                 );
+                debug!(
+                    "replayed into bank slot: {:?} / {:?}, result: {:?}, the execute_timings stats are: {:?}",
+                    bank.slot(),
+                    bank_slot,
+                    replay_result,
+                    bank_progress.replay_stats.execute_timings
+                );
+                let mut cost_model_mutable = cost_model.write().unwrap();
```

Comment:
Can we put this block into it's own function, like `update_cost_model`

Reply:
good idea ... will move it to function